src/aria/widgets/container/Tab.js

```diff
@@ -116,31 +143,31 @@ module.exports = Aria.classDefinition({
                     var selected = inside[to] === cfg.tabId;
                     var value = selected ? 'true' : 'false';
 
-                    extraAttributes.push(['aria-selected', value]);
-                    extraAttributes.push(['aria-expanded', value]);
+                    waiAttributes.push(['aria-selected', value]);
+                    waiAttributes.push(['aria-expanded', value]);
                 }
             }
 
             var value = cfg.disabled ? 'true' : 'false';
-            extraAttributes.push(['aria-disabled', value]);
+            waiAttributes.push(['aria-disabled', value]);
 
             this._updateLabelId();
 
             var id = this._getControlledTabPanelId();
             if (id != null) {
-                extraAttributes.push(['aria-controls', id]);
+                waiAttributes.push(['aria-controls', id]);
             }
-        }
 
-        var _extraAttributes = '';
-        ariaUtilsArray.forEach(extraAttributes, function (attribute) {
-            var key = attribute[0];
-            var value = attribute[1];
+            var waiAttributesString = '';
+            ariaUtilsArray.forEach(waiAttributes, function (attribute) {
+                var key = attribute[0];
+                var value = attribute[1];
 
-            _extraAttributes += ' ' + key + '="' + value + '"';
-        });
-        _extraAttributes += ' ';
-        this._extraAttributes = _extraAttributes;
+                waiAttributesString += ' ' + key + '="' + value + '"';
+            });
+            waiAttributesString += ' ';
+            this._waiAttributes = waiAttributesString;
```

Comment:
Move this to a separate method that you can call from `_widgetMarkupBegin`. It is useless to store those attributes on the instance.